### PR TITLE
High Luminosity Eyes: register signal to Move

### PR DESCRIPTION
**Pull request by larentoun**
## About The Pull Request
Now eyes "lights" update on owner's Move signal, instead of only direction.

## Why It's Good For The Game
You can have light while moving! 

## Changelog
:cl:
qol: High Luminosity Eyes' glow now moves with owner
/:cl:

### CODE REVIEW

File: code/modules/surgery/organs/eyes.dm [hunk 1]
```diff
@@ -443,10 +443,12 @@
 /obj/item/organ/internal/eyes/robotic/glow/on_insert(mob/living/carbon/eye_owner)
 	. = ..()
 	RegisterSignal(eye_owner, COMSIG_ATOM_DIR_CHANGE, PROC_REF(update_visuals))
+	RegisterSignal(eye_owner, COMSIG_MOVABLE_MOVED, PROC_REF(update_visuals))
```
Comment (san7890) [inline code/modules/surgery/organs/eyes.dm:446]:
```suggestion
	RegisterSignals(eye_owner, list(COMSIG_ATOM_DIR_CHANGE, COMSIG_MOVABLE_MOVED), PROC_REF(update_visuals))
```

much cleaner implementation of the same thing

File: code/modules/surgery/organs/eyes.dm [hunk 2]
```diff
@@ -443,10 +443,12 @@
...
 
 /obj/item/organ/internal/eyes/robotic/glow/on_remove(mob/living/carbon/eye_owner)
 	. = ..()
 	UnregisterSignal(eye_owner, COMSIG_ATOM_DIR_CHANGE)
+	UnregisterSignal(eye_owner, COMSIG_MOVABLE_MOVED)
```
Comment (san7890) [inline code/modules/surgery/organs/eyes.dm:451]:
```suggestion
	UnregisterSignal(eye_owner, list(COMSIG_ATOM_DIR_CHANGE, COMSIG_MOVABLE_MOVED))
```

`UnregisterSignal()` allows lists of multiple signals, so may as well condense this easy-peasy.

**Review by san7890 — CHANGES_REQUESTED** [review]

**Comment by san7890** [comment]
Marking this as a "draft" state PR for book-keeping purposes, feel free to hit the "Ready for Review" button down below when these reviews have been addressed in some manner.

**Comment by larentoun** [comment]
> #75040
> 
> This will become deprecated by the pr I made last week for high luminosity eyes.

Oh, cool! I'll close this one then!

**Pull request closed without merging**